src/WalkthroughElement.tsx

```diff
@@ -0,0 +1,64 @@
+import React, {FunctionComponent, ReactNode, ReactElement} from 'react';
+import Tooltip, {TooltipChildrenContext, TooltipProps} from 'react-native-walkthrough-tooltip';
+
+import {WalkthroughContext} from './ContextWrapper';
+
+type UseTooltipChildContextT =
```

Comment:
this type should have a lower case `t` at the end, right?

Reply:
the T should have indicated that it's a type, but i think without the T it's also fine as there is no name conflict